pwndbg/commands/xor.py

```diff
@@ -31,10 +31,17 @@ def xor(address, key, count):
     XOR ``count`` bytes at ``address`` with the key ``key``.
     '''
     if not isinstance(address, six.integer_types):
-        address = int(address, 16)
+        try:
+            address = int(address, 16)
```

Comment:
We should probably not use `16` here, but rather `0` so that the value is auto-detected